docs/Changelog.md

```diff
@@ -10332,6 +10332,57 @@ This version of the operator has been available since version 11 of the default
 <dd>Constrain input and output types to integer tensors.</dd>
 </dl>
 
+### <a name="CropAndResize-11"></a>**CropAndResize-11**</a>
+
+  Extracts crops from the input image tensor and resizes them using bilinear sampling or nearest neighbor sampling
+  (possibly with aspect ratio change) to a common output size specified by crop_height and crop_width.
+  Returns a tensor with crops from the input image at positions defined at the bounding box locations in boxes.
```

Comment:
Add link to the equivalent TensorFlow version.

Reply:
added